openssl-dynamic/src/main/java/io/netty/internal/tcnative/Library.java

```diff
@@ -37,7 +37,12 @@
 public final class Library {
 
     /* Default library names */
-    private static final String [] NAMES = {"netty-tcnative", "libnetty-tcnative", "netty-tcnative-1", "libnetty-tcnative-1"};
+    private static final String [] NAMES = {
+        "netty-tcnative",
+        "netty_tcnative",
+        "libnetty-tcnative",
+        "netty-tcnative-1",
```

Comment:
@normanmaurer - where is the `-1` suffix used?

we should revisit this when we move tcnative into netty (e.g. just use the `_` version)